eslint_js.yaml

```diff
@@ -0,0 +1,79 @@
+parserOptions:
```

Comment:
praise: Ah, yaml is much more reasonable of a format to use here. I didn't know eslint could do this.